app/lib/main.dart

```diff
@@ -365,6 +371,7 @@ class _EffektioHomeState extends State<EffektioHome>
             ),
             floatingActionButton: FloatingActionButton(
               onPressed: () async {
+                throw Exception('force exception');
```

Comment:
this should probably not be here.

Reply:
Sorry, my bad.
It was my test code.
Removed.